.github/auto_assign.yml

```diff
@@ -0,0 +1,35 @@
+# Set to true to add reviewers to pull requests
+addReviewers: true
+
+# Set to true to add assignees to pull requests
+addAssignees: false
+
+# A list of reviewers to be added to pull requests (GitHub user name)
```

Comment:
Add a note that engine committers should be added here upon joining the team (and maybe a link to CONTRIBUTING.md for people who want to see what that entails)

Reply:
Done, will add contributing note when this lands and the bot is installed.